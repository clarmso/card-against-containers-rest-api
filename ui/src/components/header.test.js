import React from "react"
import renderer from "react-test-renderer"
import Header from "./header"

test("Display a header without any content", () => {
  const component = renderer.create(<Header />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test("Display a header with title and description", () => {
  const title = "Prince Edward Island"
  const description = "Locals just call this seventh province The Island"
  const component = renderer.create(
    <Header title={title} description={description} />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
