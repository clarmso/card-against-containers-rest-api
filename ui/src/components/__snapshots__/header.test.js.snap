// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display a header with title and description 1`] = `
<div
  className="MuiGrid-root makeStyles-header-1 MuiGrid-item"
>
  <h1
    className="makeStyles-gameName-2"
  >
    
  </h1>
  <p
    className="makeStyles-subtitle-3"
  >
    
  </p>
</div>
`;

exports[`Display a header without any content 1`] = `
<div
  className="MuiGrid-root makeStyles-header-1 MuiGrid-item"
>
  <h1
    className="makeStyles-gameName-2"
  >
    
  </h1>
  <p
    className="makeStyles-subtitle-3"
  >
    
  </p>
</div>
`;
